### PR TITLE
Add new task

### PR DESCRIPTION
**Pull request by BeatiCode**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -31,13 +41,57 @@ function createListItem() {
...
 
-    const ul = document.getElementsByTagName("ul");
-    const li = createListItem();
+function renderTasks() {
 
-    ul[0].prepend(li);
+    if(tasksList.length > 0) {
+        const ul = document.querySelector('.todo-list');
+        let i = 0;
+        while(i < tasksList.length) {
```
Comment (agolendukhin) [inline index.js]: better to use for

File: index.js [hunk 2]
```diff
@@ -31,13 +41,57 @@ function createListItem() {
...
+        }
+    }
 
-}
+        const input = document.querySelectorAll('.toggle');
+        const label = document.querySelectorAll('.todo-list li label');
+        console.log(label);
+        
+        let i = 0;
+        while(i < tasksList.length) {
```
Comment (agolendukhin) [inline index.js]: same (for)

File: index.js [hunk 3]
```diff
@@ -31,13 +41,57 @@ function createListItem() {
...
+        }   
 
- renderTask()
\ No newline at end of file
+
+};
+
+function getid(arr) {
+
+    if(arr.length > 0){
```
Comment (agolendukhin) [inline index.js:62]: spaces. try prettier

File: index.js [hunk 4]
```diff
@@ -31,13 +41,57 @@ function createListItem() {
...
+        return arr[arr.length-1]['id'];
+    } else {
+        return arr;
+    };
+};
+
+function createNewTask() {
+
```
Comment (agolendukhin) [inline index.js]: no need in empty line

**Comment by BeatiCode** [comment]
Что то напутал))

**Pull request closed without merging**